crane/container.go

```diff
@@ -40,7 +40,7 @@ type ContainerInfo interface {
 	ActualName() string
 	Image() string
 	ID() string
-	Dependencies() *Dependencies
+	Dependencies(excluded []string) *Dependencies
```

Comment:
I wanted to have it like that, then decided against it since this makes it non-trivial for the generate feature to use `Dependencies`.